### ISSUE TITLE
IPM update procedure is not described

### ISSUE THREAD

**Issue by janihur** [intersystems/ipm#673]
I'm going to update from 0.7 to 0.9 and I don't see the update procedure described anywhere.

All I can find from the [fine manual](https://github.com/intersystems/ipm/wiki/01.-Getting-started) is:
> Import it into IRIS and compile via any desired way

What happens if I do:
```
NAMESPACE>set sc = $system.OBJ.Load("/<PATH>/zpm-0.9.1.xml","ck")
```
on top of 0.7.x installation?

This is not (yet) a show-stopper as my "update" is really a new install.

**Comment by janihur** [comment]
Related question: is importing the same version of IPM idempotent? I.e. is it always safe to run
```
NAMESPACE>set status = $system.OBJ.Load("/<PATH>/zpm-0.9.1.xml","ck")
NAMESPACE>zpm "enable -map -globally"
```
any number of times.

I'm incorporating IPM installation into our tooling and I'd like to hide all these details from the users so they can always just type:
```
$ iris-tool install ipm
```
in operating system command line and get the right IPM version defined by our tooling without nasty side-effects.

**Comment by janihur** [comment]
Not exactly upgrade related but an installation pattern documentation issue anyway. Am I right that:

> IPM supports two different installation patterns:
> * Install a version in one namespace (preferably `%SYS`) and then map the version in to any other namespace. This way all the namespaces in an IRIS instance share the same IPM version.
> * Install a version in the every namespace. This way the different namespaces can have different versions.
>
> These patterns can be mixed in the same IRIS instance, but it's probably not a good idea to mix them, YMMV.

I would like to see text like this explaining the big picture with the supporting IPM commands in the docs.

**Comment by isc-etamarch** [comment]
@janihur I've updated the wiki with some instructions regarding install. You are right, you should be able to just install a new version on top without issue.

If you ran 
```
zpm "enable -map -globally"
```

then the new install will automatically be used in the rest of the namespaces (Note that if you have a new namespace without ipm, you will have to run the command again).

In terms of mapping vs fresh installs in each namespace, I would highly recommend choosing one or the other. If you're curious,  if you run *zpm "enable -map -globally"*, and then run a separate install in another namespace, this namespace is now permanently disconnected from the global mapping. Even if you were to update zpm on the original namespace and map globally again, this new namespace will retain its own installed version of ipm.

As of 0.9.2, you can also run 
```
zpm "enable -community"
```
which will reset your ipm version to the latest version on the community registry, as well as map this version globally. This can be useful, since to update ipm would just require running the command
```
zpm "reinstall zpm"
```
and it will automatically grab the latest version of ipm from the community registry (this will also be reflected in all the other namespaces).

In short, if using "enable -map -globally", make sure you are doing all your zpm updates in the same namespaces, otherwise they won't be synced anymore.
